.github/workflows/validate_pull_request.yml

```diff
@@ -14,10 +14,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js 12.x
+    - name: Use Node.js 14.x
       uses: actions/setup-node@v1
       with:
-        node-version: '12.x'
+        node-version: '14.x'
```

Comment:
Suggestion : We can also add [engines snippet](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines) to package.json to keep a track of it better.